routing/src/vrf.rs

```diff
@@ -0,0 +1,447 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Open Network Fabric Authors
+
+//! VRF module to store Ipv4 and Ipv6 routing tables
+
+use std::hash::Hash;
+use std::net::IpAddr;
+#[cfg(test)]
+use std::str::FromStr;
+
```

Comment:
why not put this `use` under the `test` module?

Reply:
Agreed, it should go there, thanks.